instance/models/mixins/storage.py

```diff
@@ -276,6 +277,14 @@ def _create_bucket(self, attempts=4, ongoing_attempt=1, retry_delay=4, location=
                 else:
                     raise e
             bucket.set_cors(get_s3_cors_config())
+            bucket.configure_versioning(versioning=True)
```

Comment:
@Metfriet Just to be sure, this function is called for any instance upon provisioning a VM, even if the instance already existed with VMs in the past?

Reply:
@UmanShahzad, anytime the spawn_appserver method in OpenEdXInstance gets called for an instance with s3 storage this function gets called (via some other methods) and that should happen regardless of whether or not there was a VM before right?